base-image-import.yml

```diff
@@ -10,6 +10,7 @@ schedules:
   branches:
     include:
     - master
+  always: true
```

Comment:
does it only build on previously failed builds otherwise?

Reply:
Only if there are changes